src/lights/PointLight.js

```diff
@@ -10,6 +10,7 @@ THREE.PointLight = function ( color, intensity, distance ) {
 
 	this.intensity = ( intensity !== undefined ) ? intensity : 1;
 	this.distance = ( distance !== undefined ) ? distance : 0;
+	this.decayExponent = ( decayExponent !== undefined ) ? decayExponent : 1;	// for physically correct lights, should be 2.
```

Comment:
Ok. Lets leave default at `1` then.
How about renaming to `decay`? This is purely a API aesthetics thing \*blush\*.